### PR TITLE
Update Networking.md

### PR DESCRIPTION
**Pull request by jayakody**
Updating the "Big Switch Networks" (www.bigswitch.com) Kubernetes solution details.

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.9 Features: set Milestone to `1.9` and Base Branch to `release-1.9`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.
>
> NOTE: After opening the PR, please *un-check and re-check* the "Allow edits from maintainers" box. This is a temporary workaround to address a known issue with GitHub.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6537)

### CODE REVIEW

**Comment by jayakody** [comment]
I signed the CLA. Let me know if there are any other issues.

**Comment by jayakody** [comment]
Retry (Since CLA is failing)

**Comment by jayakody** [comment]
Hi @abiogenesis-now  Just wondering. Would you be able to approve these changes and add Big Switch to the cluster networking section? Please let me know if you need more input from our side.

**Comment by heckj** [comment]
👋 @jayakody - Apologies for coming back to this weeks later, I was working on handling some of the backlog PR's in queue and spotted it. I'm afraid the change needs a rebase at this point, but if you'd do that and ping me, we'll get this merged into place.

File: docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -106,7 +106,12 @@ imply any preferential status.
 ### ACI
 
 [Cisco Application Centric Infrastructure](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/application-centric-infrastructure/index.html) offers an integrated overlay and underlay SDN solution that supports containers, virtual machines, and bare metal servers. [ACI](https://www.github.com/noironetworks/aci-containers) provides container networking integration for ACI. An overview of the integration is provided [here](https://www.cisco.com/c/dam/en/us/solutions/collateral/data-center-virtualization/application-centric-infrastructure/solution-overview-c22-739493.pdf).
+
+### Big Cloud Fabric from Big Switch Networks
+ 
+[Big Cloud Fabric](https://www.bigswitch.com/container-network-automation) is a next-generation data center switching fabric using software controls and a leaf-spine Clos architecture delivered on open networking hardware.
  
+Big Cloud Fabric [integrates with the Kubernetes](https://www.youtube.com/watch?v=y46rZWz_DWc) to provide automated policy-based networking for containers in container-only and heterogeneous environments. It also enhances troubleshooting and visibility for container workloads.
```
Comment (tengqm) [inline docs/concepts/cluster-administration/networking.md]: to provide -> provides?

**Pull request merged**